components/switch/switch-mixin.js

```diff
@@ -194,6 +215,9 @@ export const SwitchMixin = superclass => class extends RtlMixin(FocusVisiblePoly
 	updated(changedProperties) {
 		super.updated(changedProperties);
 		if (!changedProperties.has('on') || changedProperties.get('on') === undefined) return;
+		/**
+		 *  Dispatched when there is a change to the switch value. "on" corresponds to the current state of the switch.
```

Comment:
Documented this in line as doing it at the top of the file didn't seem to pass it through, might be due to the fact that this is a mixin. Doing it like this makes it appear in both `switch` and `switch-visibility` variants